### DISCUSSION TITLE
Custom export file names

### DISCUSSION THREAD

**Discussion by sampie777** [SAHorowitz/MSFS2020-PilotPathRecorder#14] [Ideas]
Hi there,

Is it possible to make the export file name customizable? I'd like to have my export file names beginning with the date (YYYY-MM-DD + time format) and the plane second, so my files are automatically sorted by date. 

An even more advanced option will be to include departure region / airport, but that's just a nice to have.

**Comment by SAHorowitz** [comment]
This has been coded and is ready for release. It supports several replaceable values like plane name, flight start time, flight end time (both times support any windows date and time format), closest airport, city, state, at start of flight and end of flight. All easily configurable.  You can interweave fixed strings in the name as well.

Should release in next day or so.

**Reply by sampie777** [reply]
Wow, that's a lot. Sounds awesome! Thanks!

**Comment by SAHorowitz** [comment]
1.4.0 has been released with this functionality in it.
